src/handlers/user.ts

```diff
@@ -52,14 +54,43 @@ export const loginUser = async (req, res, next) => {
     });
 
     res.cookie('refresh_token', refreshToken, { httpOnly: true, maxAge: 24 * 60 * 60 * 1000 });
-    res.json({ accessToken });
+    res.json({ accessToken, updatedUser });
 
 
   } catch (err) {
     next(err);
   }
 };
 
+export const logoutUser = async (req, res, next) => {
+  try {
+    const userToken = await prisma.user.findUnique({
+      where: {
+        refresh_token: req.body.refresh_token
+
+      }
+    });
+
+    if (userToken) {
+      if (req.body.id) {
+        await prisma.user.update({
+          where: { id: req.body.id },
+          data: { refresh_token: null },
+        });
+      } else {
+        console.error('User ID is missing in the request body');
+      }
+    }
```

Comment:
* the above condition should be splitted, first check if (!user) {res.json(error with not found)}
* on another if check if(!req.body.id) {error}
* after that put directy the logic with update (or you can try with delete)
* we shouldn't have console.error here, just res.json and express methods